lib/scope.js

```diff
@@ -32,7 +32,10 @@ module.exports = {
     let scenario = await scope.makeSureReportPartsExist( scope );
     scenario.lines.push({ type, value });
 
+    fs.appendFileSync('log.txt', `Report:\n type: ${ type }, value: ${ value }`);
```

Comment:
This is my typo from my console log, but it'd be generous of you to fix it - I have a space between the `\n` and "type". This is what I'd love it to be:

```suggestion
    fs.appendFileSync('log.txt', `Report:\ntype: ${ type }, value: ${ value }`);
```